pkg/cover/backend/elf.go

```diff
@@ -133,7 +135,12 @@ func elfReadModuleCoverPoints(target *targets.Target, module *Module, info *symb
 	if err != nil {
 		return pcs, err
 	}
-	offset := uint64(arches[target.Arch].opcodeOffset)
+	var typ uint64
+	if target.Arch == targets.ARM64 {
+		typ = uint64(elf.R_AARCH64_CALL26)
```

Comment:
The arches map here works well to avoid switches and if's scattered everywhere. I would add `callRelocType` to Arch struct.